backend/routes/planet.routes.js

```diff
@@ -170,13 +170,13 @@ router.delete('/', async (req, res) => {
         if (!planet) {
             return res.status(404).json({
                 success: false,
-                error: `Planet with id ${id} not found`,
+                error: `Planet with id ${id} could not be found`,
             });
         }
 
         res.status(200).json({
             success: true,
-            data: planet,
+            error: 'Planet deleted',
```

Comment:
Error should be empty on status 200.

Here is an example from a SQL delete route (Data is the number of rows affected in the deletion):
```
{
	"success": true,
	"error": "",
	"data": 1
}
```